lib/hyrax/transactions/container.rb

```diff
@@ -88,7 +89,13 @@ class Container
         end
       end
 
-      namespace 'work' do |ops|
+      namespace 'work_resource' do |ops| # valkyrie works
+        ops.register 'add_file_sets' do
+          Steps::AddFileSets.new
+        end
+      end
+
+      namespace 'work' do |ops| # legacy AF works
```

Comment:
I love mentally expanding "legacy AF" into the "kids these days" vernacular.  Great comment!